nautobot/extras/views.py

```diff
@@ -1493,10 +1494,39 @@ class JobEditView(generic.ObjectEditView):
     template_name = "extras/job_edit.html"
 
 
+class JobBulkEditView(generic.BulkEditView):
+    queryset = JobModel.objects.all()
+    table = tables.JobTable
+    form = forms.JobBulkEditForm
+    template_name = "extras/job_bulk_edit.html"
+
+    def extra_post_save_action(self, obj, form):
+        cleaned_data = form.cleaned_data
+
+        # Handle text related fields
+        for overridable_field in JOB_OVERRIDABLE_FIELDS:
+            override_field = overridable_field + "_override"
+            clear_override_field = "clear_" + overridable_field + "_override"
+            reset_override = cleaned_data.get(clear_override_field, False)
+            override_value = cleaned_data.get(overridable_field)
+            if reset_override:
+                setattr(obj, override_field, False)
```

Comment:
In `JobEditForm.save()`, we would also reset `overridable_field` to the value from `obj.job_class`, I believe?

Reply:
Setting the override_field to False automatically reverts the field values to default based on my manual testing at least. Will add unit tests to confirm this behavior.